client/src/components/OrderHistory/OrderHistory.js

```diff
@@ -13,8 +13,8 @@ function OrderHistory() {
   const userID = auth.currentUser.uid;
   useEffect(() => {
     dispatch(getOrdersAsync(userID));
-    //console.log(list);
   }, []);
+  console.log(list);
```

Comment:
If this console log isn't needed anymore, it might be best to remove it

Reply:
removed